pkg/templates/charts/toggle/console/templates/acm-cli-consoleclidownload.yaml

```diff
@@ -12,15 +12,18 @@ metadata:
   name: acm-cli-downloads
 spec:
   description: |
-    With the CLIs for Red Hat Advanced Cluster Management for Kubernetes (RHACM) you 
+    With the CLIs for Red Hat Advanced Cluster Management for Kubernetes you 
     can enhance your multicluster experience at the terminal. This includes:
     - `policytools`
-        - Interact with RHACM policies locally, including resolving templates locally.
+        - Interact with Red Hat Advanced Cluster Management policies locally, 
+          including resolving templates locally and (Dev Preview) executing 
+          a ConfigurationPolicy locally.
     - `PolicyGenerator`
-        - Build RHACM policies from Kubernetes manifest YAML files, which are 
-          provided through a `PolicyGenerator` manifest YAML file that is used to 
-          configure it. The Policy Generator is implemented as a Kustomize generator 
-          plug-in, allowing integration with GitOps.
+        - Build Red Hat Advanced Cluster Management policies from Kubernetes 
+          manifest YAML files, which are provided through a `PolicyGenerator` 
+          manifest YAML file that is used to configure it. The Policy 
+          Generator is implemented as a Kustomize generator plug-in, allowing 
+          integration with GitOps.
```

Comment:
```suggestion
          integration with Red Hat OpenShift GitOps.
```

Reply:
I was intentionally using the generic "GitOps" (though most users would likely use Red Hat Openshift GitOps as their GitOps tool).

Reply:
Also, do you think there should be a doc link for the generator in here? I realized that how to use the PolicyGenerator from here is not obvious.

Reply:
Mhmm I think add a link to the doc should be fine. Just a reminder that it will appear as broken until we publish the docs